<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exp3DDlg</class>
 <widget class="QDialog" name="Exp3DDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="path">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Path:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioVRML">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export VRML</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioSTL">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export STL</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioSTLBinary">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Use STL binary format</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioSTLAscii">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Use STL ASCII format</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioObj">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Wavefront OBJ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Exp3DDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Exp3DDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
